--- origsrc/workrave-1_10_10/configure.ac	2016-03-01 20:38:31.516962700 -0600
+++ src/workrave-1_10_10/configure.ac	2016-03-01 23:03:28.089787200 -0600
@@ -893,14 +893,14 @@ config_gstreamer=no
 if test "x$enable_gstreamer" != "xno";
 then
     PKG_CHECK_MODULES([GSTREAMER],
-                      [gstreamer-0.10],
+                      [gstreamer-1.0],
                       [config_gstreamer=yes
                        AC_DEFINE(HAVE_GSTREAMER, 1, [Use GStreamer for playing sounds])], [ ])
 		     
     if test "x$config_gstreamer" != "xyes";
     then
         PKG_CHECK_MODULES([GSTREAMER],
-                      [gstreamer-1.0],
+                      [gstreamer-0.10],
                       [config_gstreamer=yes
                        AC_DEFINE(HAVE_GSTREAMER, 1, [Use GStreamer for playing sounds])],
                          [if test "x$enable_gstreamer" = "xyes"; then
