--- origsrc/workrave-1.10.1/frontend/applets/common/src/Makefile.am	2011-10-25 13:04:00.000000000 -0500
+++ src/workrave-1.10.1/frontend/applets/common/src/Makefile.am	2013-11-12 20:24:50.272861600 -0600
@@ -18,6 +18,8 @@ libworkrave_private_1_0_la_SOURCES = \
 libworkrave_private_1_0_la_CFLAGS = \
 	-Wall -std=c99 $(GTK_CFLAGS) -I$(srcdir)/../include \
 	-DWORKRAVE_PKGDATADIR="\"${pkgdatadir}\""
+libworkrave_private_1_0_la_LDFLAGS = \
+	-avoid-version -no-undefined
 libworkrave_private_1_0_la_LIBADD = \
 	$(GTK_LIBS)
 
