--- origsrc/workrave-1_10_10/common/src/Locale.cc	2015-12-12 07:10:11.000000000 -0600
+++ src/workrave-1_10_10/common/src/Locale.cc	2016-03-01 22:32:52.572706100 -0600
@@ -256,7 +256,7 @@ Locale::get_week_start()
     }
 #endif
 
-#ifdef PLATFORM_OS_UNIX
+#ifdef __linux__
   union { unsigned int word; char *string; } langinfo;
   gint week_1stday = 0;
   gint first_weekday = 1;
@@ -274,6 +274,22 @@ Locale::get_week_start()
     g_warning ("Unknown value of _NL_TIME_WEEK_1STDAY.\n");
 
   week_start = (week_1stday + first_weekday - 1) % 7;
+#elif defined PLATFORM_OS_UNIX
+  char *gtk_week_start;
+#define GTK_WEEK_START "calendar:week_start:0"
+  gtk_week_start = dgettext ("gtk30", GTK_WEEK_START);
+
+  if (strncmp (gtk_week_start, "calendar:week_start:", 20) == 0)
+    week_start = *(gtk_week_start + 20) - '0';
+  else
+    week_start = -1;
+
+  if (week_start < 0 || week_start > 6)
+    {
+      g_warning ("Whoever translated calendar:week_start:0 for GTK+ "
+                 "did so wrongly.\n");
+      week_start = 0;
+    }
 #endif
 
   return week_start;
